### ISSUE TITLE
Unable to get crypt.py to work 

### ISSUE THREAD

**Issue by cchorn** [ywangd/stash#228]
On installation I get this error:

F7DDEF7C3DE6/tmp/pycrypto-2.6.1.tar.gz (446240 bytes)
    446240  [100.00%]
Extracting archive file ...
Archive extracted.
Running setup file ...
TypeError('can only concatenate list (not "OmniClass") to list',)
Failed to run setup.py
Fall back to directory guessing ...
Error: Cannot locate packages. Manual installation required.

**Comment by ywangd** [comment]
Pycrpto has C extension code and cannot be installed using stash pip. It only works for Pure Python packages.

**Comment by bennr01** [comment]
@ywangd i think he meant the `crypt` command (`$STASH_ROOT/bin/crypt.py`), which tries to install simplecrypto using `_stash("pip install simplecrypto")`. `crypt` worked some time ago, but probably stopped working due to changes in `simplecrypto`. Maybe we can fix this by specifying the `simplecrypto` version to install?

**Comment by ywangd** [comment]
@bennr01 my bad. I thought it was pycrypto. In this case, yes we could fix it by specifying the working version. However, I looked through simplecrypto's setup file and the pycrypto dependency was added 3 years ago. So maybe it was not due to any recent changes of simplecypto but rather changes to stash pip. Earlier version of stash pip does not try execute `setup.py` file but simply extract the guessed source code folder into site-packages.
